### PR TITLE
Clean ElementMap.h

### PR DESCRIPTION
**Pull request by mosfet80**
Clean code

### CODE REVIEW

File: src/App/ElementMap.h [hunk 1]
```diff
@@ -254,6 +254,10 @@ class AppExport ElementMap: public std::enable_shared_from_this<ElementMap> //TO
 
     /// Reverse hashElementName()
     MappedName dehashElementName(const MappedName& name) const;
+
```
Comment (luzpaz) [inline src/App/ElementMap.h:257]: tailing whitespace

File: src/App/ElementMap.h [hunk 2]
```diff
@@ -262,11 +254,11 @@ class AppExport ElementMap: public std::enable_shared_from_this<ElementMap> //TO
 
     /// Reverse hashElementName()
     MappedName dehashElementName(const MappedName& name) const;
-
+
```
Comment (chennes) [inline src/App/ElementMap.h:257]:
```suggestion

```

File: src/App/ElementMap.h [hunk 3]
```diff
@@ -262,11 +254,11 @@ class AppExport ElementMap: public std::enable_shared_from_this<ElementMap> //TO
 
     /// Reverse hashElementName()
     MappedName dehashElementName(const MappedName& name) const;
-
+    
     //FIXME duplicate code? as in copy/paste
     const MappedNameRef* findMappedRef(const IndexedName& idx) const;
     MappedNameRef* findMappedRef(const IndexedName& idx);
-
+
```
Comment (chennes) [inline src/App/ElementMap.h:261]:
```suggestion

```

**Review by chennes — COMMENTED** [review]
If we're cleaning code, we definitely shouldn't add new linter warnings.

**Comment by maxwxyz** [comment]
@mosfet80 could you resolve the conflicts?

**Comment by maxwxyz** [comment]
@mosfet80 ping for conflicts

**Pull request merged**